data/language/dutch.txt

```diff
@@ -584,7 +584,7 @@ STR_0582    :
 STR_0583    :
 STR_0584    :
 STR_0585    :
-STR_0586    :Bootvormige karretjes lopen over achtbaanrails, waardoor ze hellende bochten en diepe afdalingen kunnen maken, waarna ze in bakken water neerplonzen en rustig een stukje kunnen varen
+STR_0586    :Bootvormige karretjes rijden over een achtbaanrails, waardoor ze hellende bochten en diepe afdalingen kunnen maken, waarna ze in bakken water neerplonzen en rustig een stukje kunnen varen
```

Comment:
Die verklaring van Corsleutel had ik nog niet gezien. Maar die echt onjuist: een spoor heeft twee rails. Rails is synoniem met spoorstaven, niet met spoor. Sterker nog: hij spreekt zichzelf tegen door eerst van '2 rails' en daarna van 'een (achtbaan)rails' te spreken. Hier is ook nog een bron: http://www.vandale.nl/opzoeken?pattern=rails&lang=nn

Als je zonodig 'een' wilt gebruiken, is 'rijden over een achtbaanspoor' wel correct.